bug_crowd/test.py

```diff
@@ -61,7 +61,7 @@ def test_get_api_uri_for_bounty_submissions(self):
             as expected.
         """
         expected_uri = self.client.base_uri + (
-            'bounties/%s/submissions' % url_quote(self._bounty['uuid']))
+                'bounties/%s/submissions' % url_quote(self._bounty['uuid']))
```

Comment:
Hmm why did this line change?

Reply:
Looks like my linter added the tab. I'll remove it.